packages/commonwealth/client/scripts/views/pages/view_snapshot_proposal/snapshot_information_card.tsx

```diff
@@ -41,7 +41,7 @@ const SnapshotInfoLinkRow = (props: SnapshotInfoLinkRowProps) => {
       <CWText type="caption" className="snapshot-info-row-label">
         {label}
       </CWText>
-      <a href={url} target="_blank">
+      <a href={url} target="_blank" rel="noreferrer">
```

Comment:
I don't know if we need this, but if we do, should we do it on other external links as well? If yes I suggest we do it in a different ticket.

Reply:
Hey, @mzparacha! ESLint warns that this is a security issue in older browsers.
If it's alright with you, I'd like to keep this here and make a card around adding the above to any relevant links!